brave-spark-framework-filter/src/main/java/com/github/kristofa/brave/spark/BraveSparkExceptionHandler.java

```diff
@@ -0,0 +1,49 @@
+package com.github.kristofa.brave.spark;
+
+import com.github.kristofa.brave.Brave;
+import com.github.kristofa.brave.ServerTracer;
+import spark.ExceptionHandler;
+import spark.Request;
+import spark.Response;
+import spark.utils.StringUtils;
+import zipkin.Constants;
+
+import static com.github.kristofa.brave.internal.Util.checkNotNull;
+
+public final class BraveSparkExceptionHandler implements ExceptionHandler {
+
+    private final Brave brave;
+
+    private final ExceptionHandler exceptionHandler;
+
+    public static BraveSparkExceptionHandler create(Brave brave, ExceptionHandler exceptionHandler) {

```

Comment:
I prefer hiding this type and exposing `BraveSparkFilter.exception(ExceptionHandler delegate)`

ps BraveSparkFilter name change in another comment..